playbooks/cluster/openshift/README.md

```diff
@@ -8,3 +8,4 @@ Create an OpenShift Cluster
 | kubevirt_openshift_version| 3.7| <ul><li>3.7</li><li>3.9</li></ul>|OpenShift cluster version.|
 | openshift_ansible_dir | openshift-ansible | |Path to the openshift-ansible directory.|
 | openshift_playbook_path | playbooks/byo/config.yml |<ul><li>playbooks/byo/config.yml</li><li>playbooks/deploy_cluster.yml</li></ul>|Path to the OpenShift deploy playbook. <br>3.7: **playbooks/byo/config.yml**<br>3.9: **playbooks/deploy_cluster.yml**|
+|storage_role | None | Which storage flavor should be deployed in the cluster. The supported values are: storage-glusterfs -> Deploy Gluster storage (Don't forget to add gluster nodes in the inventory file).
```

Comment:
`|storage_role | |<ul><li>storage-glusterfs</li><ul> |Storage flavor to deploy in the cluster. Don't forget to add Gluster nodes in the [inventory](https://github.com/kubevirt/kubevirt-ansible/blob/master/inventory) file).`

Reply:
I already fixed this in #176.

Reply:
right, I had a deja vu feeling =)